.github/release-drafter.yml

```
@@ -79,3 +79,21 @@ replacers:
     replace: 'Custom WAR Packager'
   - search: '@dependabot-preview'
     replace: '@dependabot'
+
+autolabeler:
+  - label: 'chore'
```

```suggestion
  - label: 'documentation'
```

Maybe Asciidoc as well

---

done, asciidoc is not present in the categories at the moment